csm/core/data/models/session.py

```diff
@@ -0,0 +1,40 @@
+# CORTX-CSM: CORTX Management web and CLI interface.
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+from csm.core.blogic.models import CsmModel
+from datetime import datetime
+from schematics.types import (StringType, DateTimeType,
+                              DictType, ListType)
+
+class SessionModel(CsmModel):
+    """ Session Model data """
+
+    _id = "_session_id"
+    _session_id = StringType()
+    _expiry_time = DateTimeType()
+    _user_id = StringType()
+    #TODO: Need to verify and Discuss
+    _permission = DictType(ListType(StringType))
```

Comment:
Hi @abhijit1patil, as we discussed, have you tried this with DictType(SetType(StringType)) ?